lazer/contracts/sui/sdk/js/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@pythnetwork/pyth-lazer-sui-js",
+  "version": "0.1.0",
+  "description": "TypeScript SDK for the Pyth Lazer Sui contract",
+  "license": "Apache-2.0",
+  "type": "module",
+  "engines": {
+    "node": "22"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/esm/client.d.ts",
+      "default": "./dist/esm/client.js"
+    }
```

Comment:
Since you're building both esm & cjs you need to export them both too:

```json
{
  "exports": {
    ".": {
      "import":  {
        "types": "./dist/esm/client.d.ts",
        "default": "./dist/esm/client.js"
      },
      "require":  {
        "types": "./dist/cjs/client.d.ts",
        "default": "./dist/cjs/client.js"
      }
    }
  }
}